### ISSUE TITLE
Navigation issues

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/derrida-django#205]
- ~Make coordinate nav start below filter section and "stick" to the top~
- [x] Headers that are links to themselves shouldn't be links
- [x] Give screen reader users a link to skip to the bottom nav, because otherwise they have to cycle through the filters or cards entirely to get there (BWH)
- [x] Pill button links on reference or intervention cards give no indication of their function, need `title` or aria attributes (or likely both) to cover screen reader use cases (BWH)

**Comment by thatbudakguy** [comment]
As far as I can tell, there's unfortunately no built-in utilities for implementing a "sticky" nav in Material Design Lite. that's usually a process that requires some javascript - not a ton, but our case might be special.

I played around with shifting the nav elements down on pages with a header, and it started to create problems with the positioning of the top search box. The way we structured the HTML isn't well-suited to implementing it as sticky, I think.

Going to push some code but leave the sticky header alone for now because I think it'd take some significant effort to get right (and probably break other things in the process).

**Comment by rlskoeser** [comment]
@thatbudakguy I agree with your assessment of the nav: current set up makes it difficult to adjust.  I'm not thrilled with the way it's currently set up, I think the corner nav container should be fixed and the links positioned absolutely within the fixed div.  I think that would make it much easier to optionally position below the banner then shift to fixed (with js to toggle a class with the different styles).  But that's not a simple conversion.

It did occur to me that there's a simpler approach to adding more than 4 margin navs: 5th and 6th get placed in the middle on left and right respectively, so we don't have to worry about flowing the first 4 conditionally or check how many primary nav items we have.

**Comment by rlskoeser** [comment]
All the other changes look good to me.
